system/security/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ if (USE_ZLIB)
   HPCC_ADD_SUBDIRECTORY (zcrypt)
 endif()
 
-HPCC_ADD_SUBDIRECTORY (digisign)
+HPCC_ADD_SUBDIRECTORY (cryptohelper)
```

Comment:
well partly because openssl security related stuff (at least in form of digisign stuff) was already in it's own lib, and partly because seemed sensible to bundle optional openssl security in separate lib.
A system can be build without USE_OPENSSL ..